ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -35,9 +35,13 @@ tests:
   steps:
     cluster_profile: azure-qe
     env:
-      ARO_WORKER_COUNT: "6"
-      ARO_WORKER_VM_SIZE: Standard_D8s_v3
+      ARO_APISERVER_VISIBILITY: Private
+      ARO_INGRESS_VISIBILITY: Private
```

Comment:
Please also test this PR with both ARO_APISERVER_VISIBILITY and ARO_INGRESS_VISIBILITY set to public to verify this does not break any existing jobs.